CMakeLists.txt

```diff
@@ -0,0 +1,962 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(graspit)
+
+####################################
+## Define build type ###############
+####################################
+if(NOT CMAKE_BUILD_TYPE)
+	set(CMAKE_BUILD_TYPE "Release" CACHE STRING
+        "Choose the type of build, options are: Debug Release"
+     FORCE)
+endif(NOT CMAKE_BUILD_TYPE)
+
+####################################
+## Define graspit variables ####### 
+####################################
+
+# main source directory
+set (GSRC ${PROJECT_SOURCE_DIR})
+
+set(PQP_COLLISION false CACHE PATH "if false, the in-build collision in graspit is used instead.")
+set(CGDB true CACHE PATH "compile CGDB sources as well")
```

Comment:
Sounds reasonable to me. When I continue with the Windows compilation test I can also add this change, if it's not been done before then (I'll probably get around to it by Wed /Thurs this week).